modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/emodelrelease/v0.1.1.json

```diff
@@ -8,8 +8,7 @@
   "@type": "nxv:Schema",
   "imports": [
     "{{base}}/schemas/neurosciencegraph/commons/entity/v0.1.0",
```

Comment:
This imports is not used. Let remove it.

Reply:
ok